src/selection.ts

```diff
@@ -0,0 +1,62 @@
+export enum SelectionTypes {
```

Comment:
> hardcoding constant no longer cause misspelling problem because if you misspell the constant

Good to know! I guess I was less worried about misspellings and more concerned about how easily constants can be refactored. Perhaps my current usage of enums is incorrect, but I can't seem to get the following code to build and the "renaming symbol" option in VSCode doesn't seem to propagate changes (e.g., `single` -> `point` or `multi` -> `list`). Perhaps I'm doing something wrong?

```
type SelectionType = 'single' | 'multi' | 'interval';
let a:SelectionType = 'single';
if (a === 'multi') {
  console.log(a);
}
```